src/gmm_utils.erl

```diff
@@ -151,6 +157,30 @@ list_other_zones() ->
 -spec batch_size() -> integer().
 batch_size() -> 5.
 
+
+-spec get_instrumentation_enabled() -> boolean().
+get_instrumentation_enabled() ->
+    case os:getenv("instrumentation") of
```

Comment:
czy zmienne środowiskowe nie powinny przypadkiem mieć nazw składających się wyłącznie z dużych liter?

Reply:
[dokumentacja](https://www.gnu.org/software/libc/manual/html_node/Environment-Variables.html): "Names of environment variables are case-sensitive and must not contain the character ‘=’. System-defined environment variables are invariably uppercase."

To by sugerowało, że nie muszą być

Reply:
dobra, zmieniłem